oneview/resource_fcoe_network.go

```diff
@@ -137,14 +190,27 @@ func resourceFCoENetworkRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("created", fcoeNet.Created)
 	d.Set("modified", fcoeNet.Modified)
 	d.Set("uri", fcoeNet.URI.String())
-	d.Set("connectiontemplateuri", fcoeNet.ConnectionTemplateUri.String())
+	d.Set("connection_template_uri", fcoeNet.ConnectionTemplateUri.String())
 	d.Set("status", fcoeNet.Status)
 	d.Set("category", fcoeNet.Category)
 	d.Set("state", fcoeNet.State)
 	d.Set("fabricuri", fcoeNet.FabricUri.String())
 	d.Set("etag", fcoeNet.ETAG)
 	d.Set("managedsanuri", fcoeNet.ManagedSanUri)
 	d.Set("scopesuri", fcoeNet.ScopesUri.String())
+
+	// reads bandwidth from connection template
+	conTemp, err := config.ovClient.GetConnectionTemplateByURI(fcoeNet.ConnectionTemplateUri)
```

Comment:
Not seeing the same thing done in datasource_fcoe_network but you have done the changes for FC. We need to be consistent across Ethernet, FC, FCOE. Can you double check?

Reply:
I have.